portfolio/src/main/webapp/style.css

```diff
@@ -151,3 +153,63 @@ nav ul li a {
 .hidden {
   display: none;
 }
+
+.aboutimg1, .aboutimg2, .aboutimg3, .aboutimg4{
```

Comment:
IMO it is better to be more descriptive with these. So instead of aboutimg1, hero-image. 

Also, I think you may be right on adding the images as background-image, but I wanted to check on the reasoning there. A lot of the CSS is unnecessary if the pictures were added directly in the HTML. Internally, we avoid background-image because of latency issues.

Reply:
will change the names thanks!

The background-image is included so that the images only encompass a finite space so that they only appear when you scroll down to that particular section. If put directly in the HTML (could you also maybe clarify what this means just to make sure we're on the same page), then I would have to find ways to make the images disappear at certain positions, which is n times harder than just making each image basically elements that I can manipulate freely. Hope that made sense haha!

tho I've def seen latency issues and was wondering if there was an even better alternative to background-image